src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Color: " + getColor() + ", " + "number: " + getNumber();
```

Comment:
you can use color instead of getColor() in the toString() method
```suggestion
    public String toString() {
        return "Color: " + getColor() + ", " + "number: " + getNumber();
```